src/components/Caption/index.css

```diff
@@ -4,7 +4,6 @@
 
 .Caption {
   display: block;
-  max-width: 36rem;
```

Comment:
Just to confirm, we don't want to have captions only as wide as the paragraphs, right?

Reply:
We may want to display Image as full width, like we do in the [Image](https://design-system.barnardos.org.uk/components/content/#image) and https://design-system.barnardos.org.uk/components/content/#video examples, however what was happening was Caption would force the constraint of max-width on the image.